apps/admin/backend/src/app.test.ts

```diff
@@ -186,18 +187,38 @@ test('getSystemSettings happy path', async () => {
   expect(systemSettingsResult).toEqual(JSON.parse(systemSettings.asText()));
 });
 
-test('getSystemSettings returns default system settings when no system settings are found', async () => {
-  const { apiClient, auth } = buildTestEnvironment();
-
-  const { electionDefinition } = electionMinimalExhaustiveSampleFixtures;
-  await configureMachine(apiClient, auth, electionDefinition);
-
-  mockSystemAdministratorAuth(auth);
+test('getSystemSettings returns default system settings when there is no current election', async () => {
+  const { apiClient } = buildTestEnvironment();
 
   const systemSettingsResult = await apiClient.getSystemSettings();
   expect(systemSettingsResult).toEqual(DEFAULT_SYSTEM_SETTINGS);
 });
 
+test('getSystemSettings failure path', async () => {
+  const { apiClient, workspace, logger } = buildTestEnvironment();
+  const { store } = workspace;
+  const {
+    electionDefinition: { electionData },
+  } = electionMinimalExhaustiveSampleFixtures;
+
+  // configure with malformed system settings data, which is the only way for
+  // this path to fail
```

Comment:
I'm surprised we don't do validation when storing system settings data

Reply:
We do validation at the `app` layer not the `store` layer, which is why I had to inject the bad data at the `store` level in this test.